auacm/app/modules/competition_manager/views.py

```diff
@@ -116,39 +119,41 @@ def update_competition_data(cid):
                 response_code=400)
 
     for problem in comp_problems:
-        session.add(CompProblem(
+        database.session.add(CompProblem(
             label=problem['label'],
             cid=competition.cid,
             pid=problem['pid']
         ))
 
-    session.flush()
-    session.commit()
+    database.session.flush()
+    database.session.commit()
     return serve_response(competition.to_dict())
 
 
 @app.route('/api/competitions/<int:cid>')
 def get_competition_data(cid):
-    competition = session.query(Competition).filter(Competition.cid == cid).\
-            first()
+    competition = database.session.query(Competition).filter(
+        Competition.cid == cid).first()
```

Comment:
No, [four space hanging indent, nothing on the first line](https://google.github.io/styleguide/pyguide.html?showone=Indentation#Indentation)